resources/assets/sass/modules/_figure.scss

```diff
@@ -106,6 +106,7 @@
 
   img {
     margin: 0 auto;
+    max-width: 50px;
```

Comment:
It's not a big deal for now since we only use the figure pattern for the sponsor logo, but it'd be nice to add this as a new modifier like `&.-medium` above so that we could have "figures" that don't constrict image size.